template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,22 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
+        purifyOptions: {
+            whitelist: ['someclass']
+        },
+      // Logs out verbose logs.
+       verbose: true,
+       // An array of file extensions for determining used classes within node_modules
+       moduleExtensions: ['.html', '.vue'],
+       // Give paths to parse for rules. These should be absolute!
+      paths: glob.sync([
+        path.join(__dirname, '../src/components/*.vue'),
```

Comment:
I'm not familiar with `glob-all` but will this match file in subdirectories too? Wouldn't it be better to do something like `path.join(__dirname, '../src/**/*.vue'),`?

I create directories for routing (`/routing/`) and vuex (`/store/`) so I don't think your globbing rules would match any files in those directories.